cpp/tests/test_abm_testing_strategy.cpp

```diff
@@ -136,7 +137,11 @@ TEST(TestTestingScheme, initAndRunTestingStrategy)
 
     ScopedMockDistribution<testing::StrictMock<MockDistribution<mio::UniformDistribution<double>>>> mock_uniform_dist;
     EXPECT_CALL(mock_uniform_dist.get_mock(), invoke)
-        .Times(testing::Exactly(2)) //only sampled twice, testing criteria don't apply to third person
+        .Times(testing::Exactly(6)) //only sampled twice, testing criteria don't apply to third person
+        .WillOnce(testing::Return(0.7))
```

Comment:
This comment needs an update. Also please document the mock values

Reply:
Similar to the one above.